src/store/actions/index.ts

```diff
@@ -24,6 +26,11 @@ export const addDifficulty = (difficulties: NetworkDifficultyEstimatedHashes = [
     difficulties
 });
 
+export const addSingleBlock = (block = []) => ({
```

Comment:
Why is this an array, it should be `{}`
```suggestion
export const addSingleBlock = (block: SingleBlock = {}) => ({
```